lte/protos/session_manager.proto

```diff
@@ -371,6 +407,8 @@ message UsageMonitoringUpdateRequest {
   RATType rat_type = 7;
   TgppContext tgpp_ctx = 8;
   EventTrigger event_trigger = 9;
+  CommonSessionContext common_context = 10;
```

Comment:
why do you need the whole commonSessionContext here?

Reply:
Yeah, maybe it's not necessary here. We can just pass the individual fields here for now. I was looking at the `CreditUsageUpdate` fields and since they mostly required a lot of the Common & the RAT specific ones I added it here too.